Send blockpms and blockchallenges status in |updateuser| (#5422)
Zarel/Pokemon-Showdown: PROTOCOL.md

> Finish logging in (or renaming) by sending: `/trn USERNAME,0,ASSERTION`
> where `USERNAME` is your desired username and `ASSERTION` is `data.assertion`.

> `|updateuser|USERNAME|NAMED|AVATAR|BLOCKPMS|SETTINGS`

Reviewer: ```suggestion
`|updateuser|USERNAME|NAMED|AVATAR|SETTINGS`
```

I assume this was a typo.

> `|updateuser|USERNAME|NAMED|AVATAR|BLOCKPMS|BLOCKCHALLENGES`

Reviewer: Hm, I would probably want to send the rest via JSON, rather than ossifying them?

Author: > I think though it would be better to make a JSON field `SETTINGS` instead of tacking on additional user settings as needed?

Agreed.

> > Your name, avatar or blocking status was successfully changed. Your username is

Reviewer: ```suggestion
> Your name, avatar or settings were successfully changed. Your username is
```
